src/nvim/api/keysets.lua

```diff
@@ -33,6 +33,18 @@ return {
   get_commands = {
     "builtin";
   };
+  create_command = {
+    "force";
+    "nargs";
+    "range";
+    "complete";
+    "luaref";
+    "count";
+    "addr";
+    "bang";
+    "bar";
+    "reg";
```

Comment:
@bfredl We can't use "register" here since it's a keyword in C. It would be nice to support "register" as the key in the options dict to match the attribute name. Is there a way to get around this?

Reply:
yes it should be doable. just need a special case in the lua code that generates the keyset structs.